src/fuzz/dummy_fuzz_test.cc

```diff
@@ -0,0 +1,148 @@
+    return false;
+  }
+  fs::path tzdataDir = originDir / "data/zoneinfo/";
+  if (setenv("TZDIR", tzdataDir.c_str(), OVERWRITE)) {
+    return false;
+  }
+  return true;
+}
+#endif
+
+const std::string server_address = "localhost:1234";
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+  #ifdef __OSS_FUZZ__
+    static bool Initialized = DoOssFuzzInit();
+    if (!Initialized) { std::abort(); }
+  #endif
+
+  Server::Options options;
+  options.server_address = server_address;
+  std::unique_ptr<Server> server = Server::Create(options);
+  if (!server) {
+    return EXIT_FAILURE;
+  }
+
+  std::cout << "Server Up, Executing Test Query" << std::endl;
+
+  try {
+    // This is the connection to the emulator.
+    ConnectionOptions emulator_connection;
+    emulator_connection.set_endpoint(server_address)
+        .set_credentials(grpc::InsecureChannelCredentials());
+
+    // First we create an instance to create our database on.
+    google::cloud::spanner::Instance instance("emulator", "emulator");
+    google::cloud::spanner::InstanceAdminClient instance_client(
+        google::cloud::spanner::MakeInstanceAdminConnection(
+            emulator_connection));
+
+    future<StatusOr<google::spanner::admin::instance::v1::Instance>>
+        instance_future = instance_client.CreateInstance(
+            google::cloud::spanner::CreateInstanceRequestBuilder(instance,
+                                                                 "emulator")
+                .SetDisplayName("emulator")
+                .SetNodeCount(1)
+                .SetLabels({{"label-key", "label-value"}})
+                .Build());
+    StatusOr<google::spanner::admin::instance::v1::Instance>
+        status_or_instance = instance_future.get();
+    if (!status_or_instance) {
+      throw std::runtime_error(status_or_instance.status().message());
+    }
+    std::cout << "Created instance [" << instance << "]\n";
+
+    // Then we create a simple database.
+    google::cloud::spanner::Database database(instance, "test-db");
+
+    auto admin_client =
+        DatabaseAdminClient(google::cloud::spanner::MakeDatabaseAdminConnection(
+            emulator_connection));
+    future<StatusOr<google::spanner::admin::database::v1::Database>> f =
+        admin_client.CreateDatabase(database, {R"""(
+          CREATE TABLE Singers (
+              SingerId   INT64 NOT NULL,
+              FirstName  STRING(1024),
+              LastName   STRING(1024),
+              SingerInfo BYTES(MAX)
+          ) PRIMARY KEY (SingerId))""",
+                                               R"""(
+          CREATE TABLE Albums (
+              SingerId     INT64 NOT NULL,
+              AlbumId      INT64 NOT NULL,
+              AlbumTitle   STRING(MAX)
+          ) PRIMARY KEY (SingerId, AlbumId),
+              INTERLEAVE IN PARENT Singers ON DELETE CASCADE)"""});
+    StatusOr<google::spanner::admin::database::v1::Database> db = f.get();
```

Comment:
Another good place to move the .get() to the previous line:

auto db_or = admin_client.CreateDatabase(...).get();